External plugins are not working since version 7.11.0

## Description

I'm trying to upgrade from version 7.10.2 to version 7.11.4 but my code is not working anymore. 
The problem is with external plugins. I'm using plugins [videojs-http-source-selector](https://github.com/jfujita/videojs-http-source-selector) and [videojs-contrib-quality-levels](https://github.com/videojs/videojs-contrib-quality-levels) to have a quality selector in the UI.

Since version 7.11.0 both plugins are not registered anymore. If I force to register them calling explicitly `videojs.registerPlugin` in my module they are registered but then I have error about non existing MenuComponent registered in the `videojs-http-source-selector` plugin.

In my module I [first import](https://github.com/openfun/marsha/blob/ddc13a03be7345a44c800f49084659600c8272d0/src/frontend/Player/createVideojsPlayer.ts#L3-L4) them, then I create an [instance of videojs](https://github.com/openfun/marsha/blob/ddc13a03be7345a44c800f49084659600c8272d0/src/frontend/Player/createVideojsPlayer.ts#L26-L44) and at this time I have an error telling me that the plugin `httpSourceSelector` does not exists.

My code was working like this in version 7.10.2 of videojs. I follow both plugins documentation to setup them.

## Additional Information
Please include any additional information necessary here. Including the following:
### versions
#### videojs
what version of videojs does this occur with? 7.11.0 and above
### plugins
are any videojs plugins being used on the page? If so, please list them below.
videojs-http-source-selector: 1.1.6
videojs-contrib-quality-levels: 2.0.9

The project is in typescript (4.1.5) and built using webpack (5.21.2)

Thanks for your help

I was working on what you ask @gkatsev, I updated first on version 7.11.7 and I have now a different error:
```
Cannot find module '@videojs/vhs-utils/cjs/resolve-url.js' from 'node_modules/video.js/dist/video.cjs.js'
```
Should I open a new issue for this case ?

--- a/package.json
+++ b/package.json
@@ -82,11 +82,16 @@
     "url": "https://github.com/videojs/video.js.git"
   },
   "dependencies": {
-    "@babel/runtime": "^7.9.2",
+    "@babel/runtime": "^7.12.5",
     "@videojs/http-streaming": "2.8.2",
+    "@videojs/vhs-utils": "^3.0.2",
     "@videojs/xhr": "2.5.1",
+    "aes-decrypter": "3.1.2",
     "global": "^4.4.0",
     "keycode": "^2.2.0",
+    "m3u8-parser": "4.7.0",
+    "mpd-parser": "0.16.0",
+    "mux.js": "5.11.0",
     "safe-json-parse": "4.0.0",
     "videojs-font": "3.2.0",
     "videojs-vtt.js": "^0.15.3"

